src/Exception/ConfigException.php

```diff
@@ -37,7 +36,7 @@ public function __construct($code = 11000, $extra = '')
         $message = sprintf(
             'Config error %s %s',
             $extra,
-            ExceptionMessages::$validationMessages[$code]
+            ExceptionMessages::$modelErrorMessages[$code]
```

Comment:
Ok, it's a small fix. Looks like It'll be better to put in different branch to save atomic changes